src/utils/StyleUtils.js

```diff
@@ -48,8 +48,28 @@ const getHoverStyles = (props :{ onClick :() => any }) => {
   return null;
 };
 
+const sizes = {
```

Comment:
This is perfectly fine for now. We can get more fancy in the future and look into other available resources for device sizes / media queries.

For now, can you please add some comments on where these values are coming from?

Reply:
uh iono they're kind of arbitrary honestly. The desktop one is probably never going to be used since the helper is `max-width` based